components/core/overlay/nz-connected-overlay.ts

```diff
@@ -36,7 +36,7 @@ export class NzConnectedOverlayDirective {
   }
 
   private updateArrowPosition(position: ConnectedOverlayPositionChange): void {
-    const originEl = this.cdkConnectedOverlay.origin.elementRef.nativeElement as HTMLElement;
+    const originEl = (this.cdkConnectedOverlay.origin as ElementRef).nativeElement as HTMLElement;
```

Comment:
Could this be another types?

Reply:
https://github.com/angular/components/blob/59002e1649/src/cdk/overlay/overlay-directives.ts#L397
https://github.com/angular/components/blob/59002e1649123922df3532f4be78c485a73c5bc1/src/cdk/overlay/position/flexible-connected-position-strategy.ts#L1138